pkg/services/auth/idimpl/signer.go

```diff
@@ -13,11 +13,15 @@ import (
 var _ auth.IDSigner = (*LocalSigner)(nil)
 
 func ProvideLocalSigner(keyService signingkeys.Service) (*LocalSigner, error) {
-	key := keyService.GetServerPrivateKey() // FIXME: replace with signing specific key
+	key, err := keyService.GetPrivateKey(context.Background(), signingkeys.ServerPrivateKeyID)
```

Comment:
Just a note: I think (in the future) it would be safer to have a separate key for this signer.

Reply:
yes It is going to. before this pr it was not possible. So will make a pr to fix it in the id signer after we get this one merged :)